app/views/RoomsListView/Header/Header.js

```diff
@@ -4,7 +4,7 @@ import {
 } from 'react-native';
 import PropTypes from 'prop-types';
 
-import TextInput from '../../../presentation/TextInput';
+import SearchHeader from '../../ShareListView/Header/Header';
```

Comment:
You shouldn't import a component from another screen.
If it's a shared component, it should be under `containers` or `presentation`.

Reply:
Okay